src/SelfServe/SelfServeTypes.ts

```diff
@@ -113,18 +130,27 @@ export interface SmartUiInput {
   disabled?: boolean;
 }
 
-export enum SelfServeNotificationType {
-  info = "info",
-  warning = "warning",
-  error = "error",
+export enum PortalNotificationType {
+  InProgress = "InProgress",
+  Success = "Success",
+  Failure = "Failure",
+}
+
+interface PortalNotification {
+  titleTKey: string;
+  messageTKey: string;
+}
+
+export interface OnSavePortalNotification extends PortalNotification {
```

Comment:
>OnSavePortalNotification [](start = 17, length = 24)

Do we really need 2 diff OnSavePortalNotification and onRefreshNotificaton? I dont think so.

Reply:
not needed anymore